lhotse/recipes/ali_meeting.py

```diff
@@ -172,12 +172,8 @@ def prepare_ali_meeting(
         validate_recordings_and_supervisions(recording_set, supervision_set)
 
         if output_dir is not None:
-            supervision_set.to_file(
-                output_dir / f"alimeeting_supervisions_{part.lower()}.jsonl.gz"
-            )
-            recording_set.to_file(
-                output_dir / f"alimeeting_recordings_{part.lower()}.jsonl.gz"
-            )
+            supervision_set.to_json(output_dir / f"supervisions_{part.lower()}.json")
```

Comment:
There are various things that we can't do so easily with json files, including splitting.
Reading json files uses much more memory, you cannot just read a subset or read items one by one.

Reply:
OK.